src/main/java/domain/Lotto.java

```diff
@@ -0,0 +1,22 @@
+package domain;
+
+
+import java.util.List;
+
+public class Lotto {
+    private final List<Integer> numbers;
+
+    public Lotto(List<Integer> lottoNumbers) {
+        this.numbers = lottoNumbers;
+    }
+
+    public int matchCount(WinningNumber winningNumber) {
+        return (int) numbers.stream()
+                .filter(winningNumber::contains)
+                .count();
+    }
+
+    public List<Integer> getLottoNumbers() {
+        return this.numbers;
```

Comment:
```suggestion
    public List<Integer> getLottoNumbers() {
        return List.copyOf(this.numbers);
```
- 복사본을 반환하여 불변성을 보장하는 것은 어떨까요?